app/src/main/kotlin/com/wire/android/ui/common/LinkifyText.kt

```diff
@@ -107,7 +108,7 @@ fun LinkifyText(
             }
             if (text is UIText.DynamicString && text.mentions.isNotEmpty()) {
                 text.mentions.forEach {
-                    if (it.length <= 0 || (it.start + it.length) >= textAsString.length) {
```

Comment:
this code was added to avoid corrupted messages with mention length is out of range of the message it self